Xamarin.Forms.Maps.Android/MapRenderer.cs

```diff
@@ -522,17 +529,14 @@ void OnMapElementCollectionChanged(object sender, NotifyCollectionChangedEventAr
 					AddMapElements(e.NewItems.Cast<MapElement>());
 					break;
 				case NotifyCollectionChangedAction.Reset:
-					if (_polylines != null)
-					{
-						foreach (var nativePolyline in _polylines)
-						{
-							nativePolyline.Remove();
-
-							var formsPolyline = GetFormsPolyline(nativePolyline);
-							if (formsPolyline != null)
-								formsPolyline.PropertyChanged -= MapElementPropertyChanged;
-						}
-					}
+					_polylines?.ForEach(p => p.Remove());
+					_polylines = null;
+
+					_polygons?.ForEach(p => p.Remove());
+					_polygons = null;
+
+					_circles?.ForEach(c => c.Remove());
```

Comment:
`.ForEach()` is not performant on Android. Can you find a way to write this with a for loop instead? Thanks!